deb/build/debian/jenkins.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Jenkins Continuous Integration Server
+After=network.target
+
+[Service]
+Type=simple
+User=jenkins
+Group=jenkins
+SyslogIdentifier=jenkins
+UMask='0002'
```

Comment:
Jenkins makes `$JENKINS_HOME/secrets` user only regardless of umask.  Note that `deb/build/debian/jenkins.default ` does not set any umask at all though it suggests `027`.